httpie/output/formatters/json.py

```diff
@@ -1,14 +1,70 @@
 import json
+from typing import Generator
 
 from ...plugins import FormatterPlugin
 
+XXSI_PREFIXES = (")]}',\n", ")]}',", 'while(1);', 'for(;;)')
```

Comment:
I added the constant to ease future prefix add.

For example, I [read](http://patorjk.com/blog/2013/02/05/crafty-tricks-for-avoiding-xssi/) that some service was using `123\n` as prefix. Such prefix would break the formatting as no heuristics could handle it: it is only valid JSON values. For instance, that response's body would be problematic without the possibility to add `123\n` prefix later:
```json
123\n"some string"
```

For more context, have a look at the doctring of `JSONFormatter._find_potential_json_bodies()`.